src/main/java/com/renatusnetwork/momentum/data/stats/StatsManager.java

```diff
@@ -942,6 +948,10 @@ public void resetSpectatorMode(PlayerStats spectatorStats)
         }
     }
 
+    public void updateSquad(PlayerStats playerStats, @Nullable Squad newSquad) {
```

Comment:
I think we can remove this and just use the objects updateSquad since it is cache only and squads data never reaches the manager level, which would be used to also have a db layer. Thoughts?

Reply:
yeah it seemed a bit redundant to have an update method in statsmanager and playerstats. added an unnecessary layer